### PR TITLE
Style homepage

### PR DESCRIPTION
**Pull request by davmcgregor**
Updated the homepage to have background image and menu

### CODE REVIEW

File: App/views/index.erb [hunk 1]
```diff
@@ -1,9 +1,31 @@
...
+  border-radius: 25px;
+  margin-top: 16% ;
 
+}
+
+</style>
+
+<div class="home-menu">
+  <h1 style="padding-bottom: 40px; font-size: 70px;">Tortoise Tourism</h1>
+  <div class="btn-group mt-10">
```
Comment (lookupdaily) [inline App/views/index.erb]: does mt-10 work? looks like boostrap spacing sizes go up to 5...

File: App/views/index.erb [hunk 2]
```diff
@@ -1,9 +1,31 @@
...
+    <form action='/users/new' method='get'>
+      <input type='submit' class="btn btn-success btn-lg mr-3" value='Sign Up'>
```
Comment (lookupdaily) [inline App/views/index.erb]: Can you try m-2 or m-3 on both buttons so that there is equal padding around both? (instead of mr-3 on just the left)

**Review by lookupdaily — COMMENTED** [review]
All looks good! Can you just double check the margin settings on elements as commented? If doesn't work feel free to resolve and I'll hit approve

**Review by lookupdaily — APPROVED** [review]

File: App/views/index.erb (hunk 2 above)
Comment (davmcgregor) [reply App/views/index.erb]: done - thanks!

File: App/views/index.erb (hunk 1 above)
Comment (davmcgregor) [reply App/views/index.erb]: changed back to 5

**Pull request merged**